lib/shared-functions.sh

```diff
@@ -19,6 +19,54 @@ function read_image_version() {
   IMAGE_VERSION_PATCH=${BASH_REMATCH[3]}
 }
 
+function read_mongo_version() {
+  local mongo_image=$(read_configuration "MONGO_IMAGE")
+  local mongo_version=$(read_configuration "MONGO_VERSION")
+  if [ -z "${mongo_version}" ]; then
+    if [[ "$mongo_image" =~ ^mongo:([0-9]+)\.(.*)$ ]]; then
+      # when running a chain of commands (example: bin/up -> bin/docker-compose) we're passing
+      # SKIP_WARNINGS=true to prevent the warning message to be printed several times
+      if [[ -z ${SKIP_WARNINGS:-} ]]; then
+        echo "-------------------  WARNING  ----------------------"
+        echo "  Deprecation warning: the mongo image is now split between MONGO_IMAGE"
+        echo "  and MONGO_VERSION configurations. Please update your config/overleaf.rc as"
+        echo "  your current configuration may stop working in future versions of the toolkit."
+        echo "  Example: MONGO_IMAGE=mongo"
+        echo "           MONGO_VERSION=6.0"
+      fi
```

Comment:
```suggestion
      echo "------------------- WARNING ----------------------"
      fi
```
Bonus: Adding a footer might make this easier to read (same for the error entries).

```
toolkit$  bin/docker-compose ps
-------------------  WARNING  ----------------------
  Deprecation warning: the mongo image is now split between MONGO_IMAGE
  and MONGO_VERSION configurations. Please update your config/overleaf.rc as
  your current configuration may stop working in future versions of the toolkit.
  Example: MONGO_IMAGE=mongo
           MONGO_VERSION=6.0
WARN[0000] toolkit/lib/docker-compose.base.yml: `version` is obsolete 
WARN[0000] toolkit/lib/docker-compose.vars.yml: `version` is obsolete 
...
```

Reply:
Agree, this is now done.